src/pyhf/probability.py

```diff
@@ -1,40 +1,54 @@
 from . import get_backend
 
 
-class Poisson(object):
+class _SimpleDistributionMixin(object):
+    def log_prob(self, value):
+        return self._pdf.log_prob(value)
+
+    def expected_data(self):
+        return self._pdf.expected_data()
+
+    def sample(self, sample_shape=()):
+        return self._pdf.sample(sample_shape)
+
+
+class Poisson(_SimpleDistributionMixin):
```

Comment:
This is in the public API and so needs docstrings. I had already written these for this PR but it seems that they got removed in the the squash that was done. :( I'll add these again, but perhaps in the future this is another reason to avoid squashing PRs into 1 commit before merging.